server/athenian/api/models/web/deployment_notification.py

```diff
@@ -66,6 +66,12 @@ def __init__(
 
     def validate_timestamps(self) -> None:
         """Post-check the request data."""
+        if self.date_finished.tzinfo is None or self.date_started.tzinfo is None:
+            raise ResponseError(
+                InvalidRequestError(
+                    "`date_finished` and `date_started` must include the timezone",
+                ),
+            )
```

Comment:
It would be nice to raise independent errors:
```
for attr in ("date_finished", "date_started"):
    if getattr(self, attr).tzinfo is None:
       raise f"`{attr}` must include the time zone"
```

Reply:
done